edumap/app/controllers/contact_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ContactController < ApplicationController
+  def new
+    @contact = Contact.new
+  end
+
+  def create
+    @contact = Contact.create(contact_params)
+    if @contact.errors.full_messages.length > 0
+      flash[:alert] = @contact.errors.full_messages[0]
+      redirect_to "/contact"
+    else
+      ActionMailer::Base.mail(:from => @contact.name + " <#{@contact.email}>", :to => "jsante@gmail.com", :subject => @contact.subject, :body => @contact.message + "\n\n" + (@contact.url ? "URL: #{@contact.url}" : "")).deliver_later
```

Comment:
Hmm, I had created a mailer for it at first, but it ended up being only one line of logic. I'd probably only move it into a mailer once there are multiple types of email which go out, otherwise you've got an extra file and unnecessary lines of code, keeping it here is dead simple.